docs: add model support (#784)
jina-ai/clip-as-service: docs/user-guides/server.md

> 

Reviewer: ```suggestion
(model-support)=
```

| Parameter               | Description                                                                                                                    |
|-------------------------|--------------------------------------------------------------------------------------------------------------------------------|

> | `name`                  | Model weights, default is `ViT-B-32::openai`. Full list of models and weights can be found at [here](#model-support)                        |

Reviewer: ```suggestion
| `name`                  | Model weights, default is `ViT-B-32::openai`. A full list of models and weights can be found {ref}`here<model-support>`                        |
```
